### PR TITLE
Remove stdout redirect

### PR DESCRIPTION
**Pull request by europaul**
We used to redirect stdout for each individual service to a file in our persistant storage, but this way these messages are not written into the logs and it's easy to forget to check the file. By not redirecting the messages we make sure all prints are available in our logs.
I didn't find any issues while manually testing this change and I made sure that the output of `fmt.Print` now lands in our logs and is further visible in Kibana.

This PR also contains a small bug fix to the following problem:
The function dumpStacks takes a file path as parameter, but was given the agent name, thus resulting in saving the stack dumps in a local file.

### CODE REVIEW

**Review by eriknordmark — APPROVED** [review]
LGTM. Thanks for fixing the dumpstacks issue - looks like that was introduced a few months back.

We should make sure that diag console output etc works as before; maybe comparing installing and booting EVE under qemu so check if the output differs?

**Comment by OhmSpectator** [comment]
@eriknordmark do you have in mind good ways to check the logs format? I can only suggest running the same scenario in pre- and post-fix environments and comparing the `logread` outputs.

Getting some kind of logging test would also prevent problems similar to the one we had when moved to a newer memlogd(?) version (the separator symbol change).

And what is exactly the "diag" you are talking about? I've heard this term several times, but never knew the strict definition 😳

File: pkg/pillar/agentlog/agentlog.go [hunk 1]
```diff
@@ -153,16 +153,17 @@ func (hook *SkipCallerHook) Levels() []logrus.Level {
...
-			}
+	agentDebugDir := fmt.Sprintf("%s/%s/", types.PersistDebugDir, agentName)
+	stacksDumpFileName := agentDebugDir + "/sigusr1"
+	memDumpFileName := agentDebugDir + "/sigusr2"
+
+	for sig := range sigs {
+		log.Functionf("handleSignals: received %v\n", sig)
+		switch sig {
+		case syscall.SIGUSR1:
+			dumpStacks(log, stacksDumpFileName)
```
Comment (rouming) [inline pkg/pillar/agentlog/agentlog.go:164]: Good catch.

**Review by rouming — APPROVED** [review]
Ack

File: pkg/pillar/agentlog/agentlog.go (hunk 1 above)
Comment (rouming) [reply pkg/pillar/agentlog/agentlog.go:164]: @europaul may I ask you to specify the commit eb0738361433 ("agentlog: add pprof debug http handler")  in the commit message which you are fixing here.

**Review by OhmSpectator — APPROVED** [review]
In general, looks fine to me. As far as I understand it should not break anything. But the question of proper testing is open.

**Comment by europaul** [comment]
@deitch do I need to bump up the pillar version in this PR? I couldn't find if it's done automatically somewhere.
Also the file `gopkgversion` contains an old version of pillar, `go.mod` of edgeview and newlog use newer versions already.

**Comment by deitch** [comment]
Nope you shouldn't. It automatically calculates the hash and injects it into eve

**Comment by OhmSpectator** [comment]
@rouming, what does the bug tag mean?

**Comment by rouming** [comment]
@OhmSpectator "bug" - the PR contains a fix for an issue, "stable" - PR should be backported to stable branches.

**Pull request merged**